demo/lib/demo_web/live/ticket_live.ex

```diff
@@ -0,0 +1,51 @@
+defmodule DemoWeb.TicketLive do
+  use Backpex.LiveResource,
+    adapter: Backpex.Adapters.Ash,
+    adapter_config: [
+      resource: Demo.Helpdesk.Ticket
+    ],
+    layout: {DemoWeb.Layouts, :admin},
+    pubsub: Demo.PubSub,
+    topic: "tickets",
+    event_prefix: "ticket_"
+
+  @impl Backpex.LiveResource
+  def singular_name, do: "Ticket"
+
+  @impl Backpex.LiveResource
+  def plural_name, do: "Tickets"
+
+  @impl Backpex.LiveResource
+  def render_resource_slot(assigns, :index, :before_page_title) do
+    ~H"""
+    <div class="alert my-4 bg-blue-100 text-sm text-blue-800">
```

Comment:
```suggestion
    <div role="alert" class="alert alert-info my-4 text-sm">
```

Reply:
I prefer `bg-blue-100` and `text-blue-800`, but we should not use hard-coded colors anymore. We could change the `alert-info` color in the config.